lib/tasks/evm_dba.rake

```diff
@@ -195,9 +179,10 @@ namespace :evm do
 
         puts "Initializing region and database..."
         # Create the region from our REGION file, initialize a new miq_database row for this region
-        EvmDba.run_cmd_with_open4("bin/rails runner 'MiqDatabase.seed; MiqRegion.seed;'")
+        AwesomeSpawn.run!("bin/rails runner", :params => ["MiqDatabase.seed; MiqRegion.seed"])
       rescue => err
-        STDERR.puts "Encountered issue setting up Database using region #{region}: #{err}\n"
+        message = err.kind_of?(AwesomeSpawn::CommandResultError) ? err.error : err.message
```

Comment:
@jrafanie I agree, but @kbrock's statement was "we should fix this in awesome spawn".  I don't think AwesomeSpawn can know that, so it leaves it to the user (which is what I did here in this change)